compiler/dyno/include/chpl/resolution/resolution-queries.h

```diff
@@ -226,6 +226,10 @@ const ResolvedFunction* resolveOnlyCandidate(Context* context,
                                              const ResolvedExpression& r);
 /**
   Compute the return/yield type for a function.
+
+  If the function returns a param, the param's value may not
+  be available. This is because the function body is not resolved when
+  the return type is explicitly declared.
```

Comment:
IMO this should compute the param, because it is part of the QualifiedType. So I'd mark this as a TODO or something like that, rather than something that seems to indicate what returnType *should* do.